src/pixelator/plot/__init__.py

```diff
@@ -2,15 +2,19 @@
 
 Copyright (c) 2023 Pixelgen Technologies AB.
 """
-
-import matplotlib.pyplot as plt
-import numpy as np
-import scipy
-from matplotlib import cm
-from typing import Literal, Tuple
-
-from pixelator.graph import Graph
-from pixelator.marks import experimental
+import os
+import tempfile
+
+os.environ["MPLCONFIGDIR"] = tempfile.mkdtemp()
+if True:  # noqa: E402
```

Comment:
Why the `if` here?

Reply:
To put the `# noqa` comment in a single line. Otherwise, you need to put it in all imports that come after. But there's a better way to do this rather than tagging all imports, happy to hear.